cdap-docs/application-templates/source/etl/templates/sinks/real-time/cube.rst

```diff
@@ -16,3 +18,34 @@ sink.
 
 To configure transformation from a StructuredRecord to a CubeFact, the mapping
 configuration is required, following StructuredRecordToCubeFact documentation.
```

Comment:
looks like this is an internal class... shouldn't be referenced here at all. Will edit this.